lib/matplotlib/__init__.py

```diff
@@ -1297,15 +1297,15 @@ def __exit__(self, exc_type, exc_value, exc_tb):
         self.__fallback()
 
 
-def use(arg, warn=False, force=True):
+def use(backend, warn=False, force=True):
     """
-    Set the matplotlib backend to one of the known backends.
+    Select the matplotlib backend for rendering.
 
     Parameters
     ----------
-    arg : str
-        The backend to switch to.  This can either be one of the
-        'standard' backend names:
+    backend : str
+        The backend to switch to.  This can either be one of the standard
+        backend names:
```

Comment:
standard backend names, which are case-insensitive: ... and remove the note below